### ISSUE TITLE
Calibre dont have enough fonts to show non latin language

### ISSUE THREAD

**Issue by ikonome** [aptalca/docker-rdp-calibre#29]
Hi, I was glade when I saw this project and found it could help me a lot with my books.

But when I opened my existing library, I found out that this docker image doesnt contains enough fonts to show non latin language such as Chinese and Japanese. I know it's not quite usefull for you to ship lots of unnessary CJK fonts taking too much place, but can you consider providing a folderpath or Variables for me to overwrite the default one so I can use my own fonts?
![chrome_2017-11-08_23-24-32](https://user-images.githubusercontent.com/1274005/32557109-2f0e5d08-c4dc-11e7-8b2e-59b5e9467d0e.png)

**Comment by aptalca** [comment]
Hi, I'm trying to understand if this is a docker image issue or an issue with calibre. For the former, I can try to fix, for the latter, I would advise contacting the developers of calibre.

Can you install calibre on a separate machine directly (without docker) and see if the fonts are displayed correctly?

I'm not sure if those fonts are supported by the OS or the app.

Thanks

**Comment by ikonome** [comment]
yes, the calibre on my windows works ok, can show CJK figures charactors correctly. 
as shown bellow,
![calibre_2017-11-09_21-30-33](https://user-images.githubusercontent.com/1274005/32608097-24d71a68-c596-11e7-98c2-c5ff16ee210c.png)

due to lack of relevent knowledge, I am not sure if it's only font problem or the system itself lacking relevent codepages, 
but I am certain that all the fonts I can see from the docker-rdp-calibre setting, none of them contains CJK characters, or say all of them are for latin charactors only. and on my own computer I can see many many more fonts.

![chrome_2017-11-09_21-35-03](https://user-images.githubusercontent.com/1274005/32608107-30733668-c596-11e7-89d4-13b9e4084229.png)

**Comment by Daopr** [comment]
@ikonome actually, this is the problem of English Linux System, the origin system does not contain any Chinese fonts. So we should install it by ourselves .

here is my Solution 

enter your container with bash 
 #(p.s. rdp-calibre is my container name replace it with yours )
`docker exec -it rdp-calibre bash`

```
apt-get install -y --force-yes --no-install-recommends fonts-wqy-microhei
apt-get install -y --force-yes --no-install-recommends ttf-wqy-zenhei
```

#now we exit the container 
`exit`
#then reboot the container with command 
`docker restart rdp-calibre`

now everything is OK .

references:
https://segmentfault.com/a/1190000005640352

**Comment by DrDoa** [comment]
@ikonome if you do not know how to do with it， you could simply search dochurch in  docker hub and use my fork-mod image 
[dochurch/docker-rdp-calibre](https://registry.hub.docker.com/u/dochurch/docker-rdp-calibre/)
